tests/test_RBAC.py

```diff
@@ -45,11 +46,12 @@ def test_encrypt_request_payload(consumer_wallet, publisher_wallet, monkeypatch)
         "compression": "zip",
     }
     req = {
-        "document": json.dumps(document),
+        "data": [document],
```

Comment:
we shhould definitely still support the version with json.dumps, this should be fully compatible.

Reply:
this is still a thing, I'll add a generic code review comment explaining what I recommend.